src/snapred/backend/recipe/algorithm/FitMultiplePeaksAlgorithm.py

```diff
@@ -28,22 +29,22 @@ def PyExec(self):
         crystalInfo = fitPeakIngredients.CrystalInfo
         peakType = fitPeakIngredients.PeakType
 
-        beta_0 = instrumentState.gsasParameters.beta[0]
-        beta_1 = instrumentState.gsasParameters.beta[1]
-        FWHMMultiplierLeft = instrumentState.fwhmMultiplierLimit.minimum
-        FWHMMultiplierRight = instrumentState.fwhmMultiplierLimit.maximum
-        L = instrumentState.instrumentConfig.L1 + instrumentState.instrumentConfig.L2
+        instrumentState.gsasParameters.beta[0]
+        instrumentState.gsasParameters.beta[1]
+        instrumentState.instrumentConfig.L1 + instrumentState.instrumentConfig.L2
```

Comment:
if these values are not used please remove them.  it looks like the formatter removed the assignments which is indicative of this.

Reply:
Done.